Code.gs

```diff
@@ -83,7 +83,8 @@ function Install(){
   if(validFrequencies.indexOf(howFrequent) == -1)
     throw "[ERROR] Invalid value for \"howFrequent\". Must be either 1, 5, 10, 15, or 30";
 
-  ScriptApp.newTrigger("main").timeBased().everyMinutes(howFrequent).create();
+  ScriptApp.newTrigger("main").timeBased().after(1000).create(); // Runs the script right after installing
```

Comment:
Why do we need a trigger for this? Is there a problem with just calling `main();`?

Reply:
It's more about being user friendly with those who don't understand how Script Apps, or coding in general, works.

If the calendar has lots of events, the installation itself may take too long. Calling `main();` does the same thing as with trigger, but the "Running function install..." will not disappear unless it has finished syncing.
With triggers, it disappears quickly and the function main will be "running in the background".

We know that, effectively speaking, it's the same but the latter allows the user to close the tab without worries that it will interrupt the installation process, even though it wouldn't interrupt anyway (as far as I know).

Reply:
Ah, right. It's a way to do "async" in Google Apps Script. Good point